lib/hardware_info.py

```diff
@@ -77,6 +78,9 @@ def read_directory_recursive(directory):
     [rpwr, 'Hardware Model', '/usr/bin/hostnamectl', r'Hardware Model:\s*(?P<o>.*)'],
     [rpwr, 'Docker Info', 'docker info', r'(?P<o>.*)', re.IGNORECASE | re.DOTALL],
     [rpwr, 'Docker Version', 'docker version', r'(?P<o>.*)', re.IGNORECASE | re.DOTALL],
+    [rpwr, 'Docker Containers', 'docker ps -a', r'(?P<o>.*)'],
+    [rpwr, 'Installed System Packages', 'if [ -f /etc/lsb-release ]; then dpkg -l | cat ; elif [ -f /etc/redhat-release ]; then dnf list installed | cat; fi', r'(?P<o>.*)', re.IGNORECASE | re.DOTALL],
```

Comment:
Why are you using `| cat` here?

Reply:
happy to take recommendations here. `dpkg -l ` gives me an `ncurses` window when I call it unpiped. This is why I feed it to cat just to get an output.

This test was only on the command line actually. I did not test if the implied pipe to Python is sufficient. Will post an update

Reply:
I didn't check the ubuntu one only the fedora command. Here you don't need the cat